lib/stoneacre_hq/prospect.rb

```diff
@@ -8,7 +8,12 @@ def initialize
       @conn = Faraday.new(
         :url => (StoneacreHq.config[:production] ? PRODUCTION_URL : DEVELOPMENT_URL ),
         :ssl => { :verify => StoneacreHq.config[:production] }
-      ) 
+      ) do |builder|
```

Comment:
@sebastianGitDL One additional improvement that you can make , is to make these classes to inherit from other class that has this initializer, which is the same everywhere.